config/log/config.go

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright 2021 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package log
+
+import (
+	"github.com/crossplane-contrib/terrajet/pkg/config"
+
+	"github.com/crossplane-contrib/provider-jet-azure/apis/rconfig"
+	"github.com/crossplane-contrib/provider-jet-azure/config/common"
+)
+
+const groupLog = "loganalytics"
+
+// Configure configures loganalytics group
+func Configure(p *config.Provider) {
+	p.AddResourceConfigurator("azurerm_log_analytics_workspace", func(r *config.Resource) {
+		r.Kind = "Workspace"
+		r.ShortGroup = groupLog
```

Comment:
Do all related resources for the "OperationalInsights" service have Terraform resource names starting with "azurerm_log_analytics"? Could you please make a quick check?

Reply:
Hi @ulucinar I checked the related resources. All "OperationalInsights" service resources have Terraform resource names starting with "azurerm_log_analytics"